### ISSUE TITLE
Remote Server Error (unauthorised)

### ISSUE THREAD

**Issue by ugarbe** [OfficeDev/office-js-docs-pr#2707]
Hello,
when testing the code, one needs to authenticate to 'Connect to the Exchange email account:
        [ ] In the "Connect to Exchange email account" dialog box, entered user name (e.g. username@domain*.onmicrosoft.com) and password of the account I created in dev program subscription setup

When I do this, I get a remote server error, (401), unauthorised.

I tried the test with Visual Studio, and use email account from the developper subscription.

Can you help,
Thanks and best regards,
Uli
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 930ce4fc-c8b8-f756-f03f-a716f1ef7883
* Version Independent ID: 753a2e38-9430-25bb-5409-b7eed9c88e47
* Content: [Build your first Outlook add-in - Office Add-ins](https://docs.microsoft.com/en-us/office/dev/add-ins/quickstarts/outlook-quickstart?tabs=visualstudio)
* Content Source: [docs/quickstarts/outlook-quickstart.md](https://github.com/OfficeDev/office-js-docs-pr/blob/master/docs/quickstarts/outlook-quickstart.md)
* Product: **outlook**
* Technology: **add-ins**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by AlexJerabek** [comment]
Thanks for reporting this @ugarbe. 

@ElizabethSamuel-MSFT, can you please investigate?

**Comment by ElizabethSamuel-MSFT** [comment]
@ugarbe Thanks for asking about this. I hit the same issue and, per the [Start the project section of the Debug Office Add-ins in Visual Studio article](https://docs.microsoft.com/office/dev/add-ins/develop/debug-office-add-ins-in-visual-studio#start-the-project), you have a couple of options.

1. Use a Microsoft account instead (e.g., outlook.com).
2. Set "Use multi-factor auth" to True on the add-in's project Deployment properties. (Click/Select the project that contains the manifest to see the Project Properties.) In my case, I also set the "Start Action" to "Microsoft Edge", "Email Address" to my M365 dev subscription user ID. When I started debugging and submitted my user ID and password in the "Connect to Exchange email account" dialog, an auth dialog popped up. After log in was successful, Outlook opened in the browser and sideloaded the add-in.

    ![image](https://user-images.githubusercontent.com/28541963/119553107-28888100-bd50-11eb-81db-12d86b0a1d3f.png)

I'll update the quick start to note the second option as well.

Let us know if this info helped you.

Thanks.
